src/julia.h

```diff
@@ -615,8 +630,7 @@ static inline void gc_wb_back(void *ptr) // ptr isa jl_value_t*
 #define jl_gc_unpreserve()
 #define jl_gc_n_preserved_values() (0)
 
-#define allocb(nb)    malloc(nb)
-DLLEXPORT jl_value_t *allocobj(size_t sz);
+#define allocb(nb) malloc(nb)
```

Comment:
It was declared twice:

https://github.com/JuliaLang/julia/blob/b458ea3937f6bfa7b84a1eb6769f487271664f60/src/julia.h#L563
https://github.com/JuliaLang/julia/blob/b458ea3937f6bfa7b84a1eb6769f487271664f60/src/julia.h#L619

Since it only needs to be declared once, I deleted one of the declarations.